src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js

```diff
@@ -89,20 +85,27 @@ export default function getEligibilityMessage({
       </>
     );
   } else if (requestReason === ELIGIBILITY_REASONS.noRecentVisit) {
-    const monthRequirement = settings?.request?.patientHistoryDuration
-      ? (settings.request.patientHistoryDuration / 365) * 12
-      : '24';
-    title = 'We can’t find a recent appointment for you';
+    const contact = facilityDetails?.telecom?.find(
+      tele => tele.system === 'phone',
+    )?.value;
+    title = 'We couldn’t find a recent appointment for you';
```

Comment:
Hmm is this the right block to modify? The linked ticket mentioned:
>Scenario:
> - DS is enabled
> - Facility requires a previous encounter
> - AND User has no previous encounters
> - AND Requests are disabled (VATS: notSupported)

Which sounds like the block from lines 18-46 should be changed instead.

Reply:
Response from Melissa: 

Use this title for both: "You can't schedule an appointment online at this facility"

So, closing this PR.